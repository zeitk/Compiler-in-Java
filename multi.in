"String" #Comment
1234"String test whitespace"
1234 #Comment
1234 Identifier
1234 Identifier"String"#Comment
void_Identifier1234
"String" "Second string" _Identifier = 1234 #Comment
#"Only a \0 comment" + _notidentifier = 1234
