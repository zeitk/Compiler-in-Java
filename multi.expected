"String"
1234
"String test whitespace"
5678
910
Identifier
1234
Identifier
"String"
void_Identifier1234
"String"
"Second string"
_Identifier
=
1234
